lib/brainstem/concerns/controller_dsl.rb

```diff
@@ -71,40 +68,96 @@ def nodoc!
           configuration[brainstem_params_context][:nodoc] = true
         end
 
+        #
+        # Specifies which presenter is used for the controller / action.
+        # By default, expects presentation on all methods, and falls back to the
+        # class derived from +brainstem_model_name+ if a name is not
```

Comment:
Should `name` be `target_class`?

> class derived from +brainstem_model_name+ if a target_class is not

Reply:
`name` is the correct word. You can override the `brainstem_model_name` with any custom name